.rubocop_todo.yml

```diff
@@ -23,14 +23,6 @@ Rails/OutputSafety:
   Exclude:
     - 'lib/currency_select.rb'
 
-# Offense count: 1
-# Cop supports --auto-correct-all.
-# Configuration parameters: Include.
-# Include: **/Rakefile, **/*.rake
-Rails/RakeEnvironment:
```

Comment:
Looks like we need to disable this rule. (The `environment` task is not available.)